taskmap/taskmap_test.py

```diff
@@ -350,6 +365,23 @@ def test_async_error_handling():
     assert graph_parallel.results['e'].args == expected['e'].args
     assert graph.results['control'] == 5
 
+    try:
+        graph = taskmap.create_graph(funcs.copy(), dependencies.copy())
+        graph = taskmap.run_async(graph, sleep=.001, trap_exceptions = False)
+    except Exception:
+        pass
+    else:
+        assert False
+
+
+    try:
+        graph_parallel = taskmap.create_graph(funcs.copy(), dependencies.copy())
+        graph_parallel = taskmap.run_parallel_async(graph_parallel, nprocs=2, sleep=.001, trap_exceptions=False)
+    except Exception:
```

Comment:
Can you assert more specifically on the exception you're expecting?  I'm not convinced by this test - it could just be that you're trying to terminate a process you can't terminate or something like that.

Please see the following for making more detailed assertions about exceptions:
https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions